libs/remote_storage/src/lib.rs

```diff
@@ -203,6 +204,91 @@ pub fn path_with_suffix_extension(original_path: impl AsRef<Path>, suffix: &str)
         .with_extension(new_extension.as_ref())
 }
 
+impl RemoteStorageConfig {
+    pub fn from_json_string(s: String) -> anyhow::Result<RemoteStorageConfig> {
```

Comment:
Let's add a comment that this should be gone once safekeeper and pageserver conf is unified.